assets/rules/defaultBlockRules.js

```diff
@@ -0,0 +1,732 @@
+      }
+    },
+    {
+      "id": 500,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*aggmaal*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 501,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*mmsbee*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 502,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*desi52*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 503,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*crazy18movies*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 504,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*xxx*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 505,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*ymlporn7*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 506,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*porn*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 507,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*xnxx*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 508,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*goindian2*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 509,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*webmaal*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 510,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*goinidian2*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 511,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*desixxxtube2*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 512,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*anybunny*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 513,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*pornolienx*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 514,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*porndor*/",
```

Comment:
We can just add the check on keyword `Porn` and it should block all sites containing that string. we don't need to separately add all domains, that list would be very big

Reply:
Yeah, although it is there, it is blocking all the domains with the keyword, but for more security added more keywords too

Reply:
lets remove the extra ones which would be handled anyway. we have a limit on number of rules as well so lets add them conviously